_component_design/popovers.md

```diff
@@ -5,90 +5,180 @@ secondary: Editing
+- Always have the popover protrude toward the center of the screen.
+- Do not hide important information with the popover.
+- The popover content should be repositioned so that it doesn't bleed off the screen.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Closeable popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Pointer specifications and pointer placement variations"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-pointer-size-placement.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Closeable popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+## Types of popovers
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Small create or edit popover
+
+A popover may be used for small create/edit workflows that need to remain in-context.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover with a small create workflow"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-small-create-edit-popover.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Closeable popover w/ header"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable-header.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Large hidden content
+
+A popover may be used to reveal hidden content, so the information is still accessible, but doesn’t clutter the screen.
+
+In this instance popover is preferred over a [tooltip]({{site.baseurl}}/components/tooltip.html) if the content is dynamic or interactive (for example, when presenting the user a table or visualization). 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"An interactive informational popover."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-large-hidden-content.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Closeable popover w/ header specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-closeable-header-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading
+
+A simple loading circle should be used to load a popover. Refer to the [Loading]({{site.baseurl}}/components/loading.html) pattern for more detailed specifications.
```

Comment:
Use a simple loading circle to load a popover.

Reply:
updated.